internal/utils/validation.go

```diff
@@ -198,3 +198,23 @@ func ValidateURLScheme(link string, scheme ...string) error {
 
 	return nil
 }
+
+// ValidateNoHTMLNorJSNorCSS detects HTML, <script> tags, inline JavaScript, and CSS styles in a string
+func ValidateNoHTMLNorJSNorCSS(input string) error {
+	// Regular expressions to catch HTML tags, <script> tags, javascript: URIs, <style> tags, and inline style attributes
+	htmlPattern := regexp.MustCompile(`</?[a-z][\s\S]*>`)
+	scriptTagPattern := regexp.MustCompile(`(?i)<script[\s\S]*?>[\s\S]*?</script>`)
```

Comment:
Given that you already have `htmlPattern := regexp.MustCompile(`</?[a-z][\s\S]*>`)`, isn't it redundant to add the script tag ? Given that it would already be caught by htmlPattern.

Reply:
Thanks for pointing that out! I removed redundancy in https://github.com/stellar/stellar-disbursement-platform-backend/pull/459/commits/2f645df6287cc404e63d8602d42229f701391a9e